### PR TITLE
Add git aliases to show sorted branches list.

### PR DESCRIPTION
**Pull request by OroianRares**
- add aliases to sort git branch results ascending/descending by branch creation time.

### CODE REVIEW

File: doc/bash/git_aliases.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 ### Git branch ###
 - **gb**: `git branch -v`
+- **gba**: `git for-each-ref --sort=committerdate refs/heads/ --format="%(authordate:short) %(color:red) (objectname:short) %(color:yellow)%(refname:short)%(color:reset) (%(color:green)%(committerdate:relative)%(color:reset))"`
```
Comment (stocarul) [inline doc/bash/git_aliases.md]: Please add a short description of the command and remove the actual command. The command is too long and I think it doesn't make sense to be in the documentation file.

Comment (stocarul) [reply doc/bash/git_aliases.md]: By the way, what is the difference between `gba` and `gbd`? I wasn't able to spot any difference in the commands.

Comment (OroianRares) [reply doc/bash/git_aliases.md]: The difference is at the sort order: 
`gba`: --sort=committerdate
`gbd`: --sort=-committerdate

The `-` sign says the sort will be descending on the specified column.

**Comment by OroianRares** [comment]
@stocarul Updated!

**Review by stocarul — APPROVED** [review]
Awesome 💯

**Pull request merged**